docs/ha-mode.md

```diff
@@ -74,6 +74,12 @@ loadbalancer_apiserver:
   port: 8383
 ```
 
+  Note: The default kubernetes apiserver configuration binds to all interfaces,
```

Comment:
it seems that it defaults to 127.0.0.1, not 0.0.0.0?

Reply:
That's what I thought, but if you deploy and check netstat you'll see it's listening on all interfaces, and also you can't start haproxy because it's already bound the port on the VIP - so I think even with localhost loadbalancing the API ends up listenining on the external interfaces?

Reply:
right, It's `kube_apiserver_insecure_bind_address` what defaults to 127.0.0.1 . so it should be ok to use this wording.